docs/add-ons/testing.md

```diff
@@ -2,15 +2,27 @@
+The fastest and recommended way to develop add-ons is using a local Visual Studio Code dev environment. The [Official Add-ons][hassio-addons] repository includes a dev container setup for VS Code which will run Supervisor and Home Assistant, with all of the add-ons mapped as Local Add-ons inside, making it simple for add-on developers on Windows, Mac and Linux desktop OS-es. 
 
-Once running, you'll need to run the task (Terminal -> Run Task) 'Start Home Assistant', which will bootstrap Supervisor and Home Assistant. You'll then be able to access the normal onboarding process via the Home Assistant instance at `http://localhost:8123/`.
+- Follow the instructions to download and install the [Remote Containers][remote-containers] VS Code extension.
+- Copy the `.devcontainer` folder from [Official Add-ons][hassio-addons] repository into the root of your add-ons folders.
+- Open the root folder inside VS Code, and when prompted re-open the window inside the container (or, from the Command Palette, select 'Rebuild and Reopen in Container'). 
+- When VS Code has opened your folder in the container (which can take some time for the first run) you'll need to run the task (Terminal -> Run Task) 'Start Home Assistant', which will bootstrap Supervisor and Home Assistant. 
+- You'll then be able to access the normal onboarding process via the Home Assistant instance at `http://localhost:8123/`.
+- The add-on(s) found in your root folder will automatically be found in the Local Add-ons repository.
 
-The add-on(s) under development will be automatically found in the Local Add-ons repository.
+For standalone add-ons, there also exists an [addon devcontainer template][hassio-addon-devcontainer] on GitHub which provides the same boilerplate dev container for new add-on projects.
 
 [hassio-addons]: https://github.com/home-assistant/hassio-addons
 [remote-containers]: https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers
-[hassio-addon-devcontainer]: https://github.com/issacg/hassio-addon-devcontainer
+[hassio-addon-devcontainer]: https://github.com/issacg/hassio-addon-devcontainer	
+
+:::info
+
+The bootstrap script (start_supervisor.sh) currently does not support Docker on Windows using the WSL 2 based engine. The message "_Timeout while waiting for docker to come up_" will appear in the terminal.
```

Comment:
```suggestion
The bootstrap script (`start_supervisor.sh`) currently does not support Docker on Windows using the WSL 2 based engine. The message "_Timeout while waiting for docker to come up_" will appear in the terminal.
```
The underscore was confusing the markdown and causing the wrong parts to be italicised

Reply:
Used proper escaping now. Using "code" tags makes that filename visually to important. I had tried that before but changed it.

![image](https://user-images.githubusercontent.com/352626/97438866-b240af00-1925-11eb-8552-a406b98e1932.png)

Reply:
Normally file names are included in code ticks elsewhere in the docs but in the note blocks maybe it doesn't work so well?

Reply:
It would look like this.
Also removed the parentheses here, because <code> style really looks terrible with them.
Your call. 



![image](https://user-images.githubusercontent.com/352626/97459427-0fdff600-193c-11eb-95dd-5c1b6932edb5.png)